ckanext/rating/utils.py

```diff
@@ -0,0 +1,35 @@
+import ckan.plugins as p
+import ckan.model as model
+import ckan.logic as logic
+from ckan.lib.base import h
+from ckan.common import request, _
+import ckan.lib.base as base
+from ckanext.rating.model import init_tables
+
+c = p.toolkit.c
+flatten_to_string_key = logic.flatten_to_string_key
+NotAuthorized = logic.NotAuthorized
+abort = base.abort
+
+def submit_package_rating(package, rating):
+    context = {'model': model, 'user': c.user or c.author}
```

Comment:
In the prev. codebase normally there are multi line comments at every function definition to document the function